csdp/hybrid/basic/apps/internal-router/internal-router.deploy.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: internal-router
+spec:
+  selector:
+    matchLabels:
+      app: internal-router
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: internal-router
+    spec:
+      containers:
+        - name: nginx
+          # do not upgrade to 1.23 until it's really critical
+          # 1.23 removed support for "Transfer-Encoding: identity" header value
+          # and it will break SSE from app-proxy
+          image: nginx:1.22-alpine
```

Comment:
This image is not good, it will give us pull rate limit from docker in hosted
need to ask ziv's team to replace with some quay.io image